integration-tests/validator/src/main/resources/application.properties

```
@@ -15,4 +15,5 @@
 ## limitations under the License.
 ## ---------------------------------------------------------------------------
 
-quarkus.native.additional-build-args =-H:ResourceConfigurationFiles=resources-config.json
\ No newline at end of file
+quarkus.native.additional-build-args =-H:ResourceConfigurationFiles=resources-config.json
```

I see that this is an old code, but we could perhaps use this occasion to change it towards Quarkus way of doing things:
```suggestion
quarkus.native.resources.includes = *.xsd
```